core/src/main/scala/scalacss/package.scala

```diff
@@ -139,4 +167,11 @@ package object scalacss {
     * @param frames Frame definitions.
     */
   final case class Keyframes(name: KeyframeAnimationName, frames: Seq[(KeyframeSelector, StyleA)])
+
+  /**
+    * Font face declaration
+    *
+    * http://www.w3schools.com/cssref/css3_pr_font-face_rule.asp
+    */
+  final case class FontFace(fontFamily: String, src: Seq[String], fontStretch: Value, fontStyle: Value, fontWeight: Value, unicodeRange: UnicodeRange)
```

Comment:
Because this is a bit long can you put on multiple lines like you did for `CssFontFace`